Bottom: 8a545618359cf175a6e8f10666f7cc6d4c1bc278
Top:    f876a45f283ff11c613aa49504a82472ba352f98
Author: Parth Shah <parth@linux.ibm.com>
Date:   2019-07-09 11:16:38 +0530

sched/topology: Introduce sched_domain_cores()


---

diff --git a/include/linux/sched/topology.h b/include/linux/sched/topology.h
index 57c7ed3fe465..cf041c6ec4ba 100644
--- a/include/linux/sched/topology.h
+++ b/include/linux/sched/topology.h
@@ -72,6 +72,8 @@ struct sched_domain_shared {
 	atomic_t	ref;
 	atomic_t	nr_busy_cpus;
 	int		has_idle_cores;
+
+	unsigned long core_mask[0];
 };
 
 struct sched_domain {
